nms/helmchart/nms/README.md

```diff
@@ -0,0 +1,113 @@
+| `manifests.deployment` | Enable Magmalte deployment | `true` |
+| `manifests.service` | Enable Magmalte services. | `true` |
+| `env.api_host` | orc8r proxy endpoint. | `[]` |
+| `env.host` | Host to bind | `[]` |
+| `env.port` | Magmalte service port bind. | `{}` |
+| `env.mapbox_access_token` | Mapbox Access token. | `` |
+| `env.mysql_host` | MySQL host IP/Name. | `` |
+| `env.mysql_db` | NMS Database name. | `` |
+| `env.mysql_user` | NMS Database user. | `` |
+| `env.mysql_pass` | NMS Database password. | `` |
+| `labels.node_selector_key` | Target Node selector label Key. | `` |
+| `labels.node_selector_value` | Target Node selector label value. | `` |
+| `image.repository` | Repository for NMS Magmalte image. | `nil` |
+| `image.tag` | Tag for NMS Magmalte image. | `latest` |
+| `image.pullPolicy` | Pull policy for NMS Magmalte image. | `IfNotPresent` |
+| `service.type` | Service type for magmalte | `ClusterIP` |
+| `service.http.port` | Service port number for magmalte | `8081` |
+| `service.http.targetport` | Service targetport number for magmalte | `8081` |
+| `service.http.nodePort` | Service nodePort number | `""` |
+| Nginx:       |
+| `manifests.configmap` | Enable Nginx configmap to store config files. | `true` |
+| `manifests.secrets` | Enable Magmalte secrets to store mysql info. | `true` |
+| `manifests.deployment` | Enable Magmalte deployment | `true` |
+| `manifests.service` | Enable Magmalte services. | `true` |
+| `labels.node_selector_key` | Target Node selector label Key. | `` |
+| `labels.node_selector_value` | Target Node selector label value. | `` |
+| `image.repository` | Repository for NMS Nginx image. | `nil` |
+| `image.tag` | Tag for NMS Nginx image. | `latest` |
+| `image.pullPolicy` | Pull policy for NMS Nginx image. | `IfNotPresent` |
+| `service.type` | Service type for Nginx | `ClusterIP` |
+| `service.http.port` | Service port number for Nginx | `443` |
+| `service.http.targetport` | Service targetport number for Nginx | `443` |
+| `service.http.nodePort` | Service nodePort number | `""` |
+| Global: |
+| `pod.replicas.nginx.server` | Number of instances to deploy for Nginx server. | `1` |
+| `pod.replicas.magmalte.server` | Number of instances to deploy for Magmalte server. | `1` |
+| `pod.resources.enabled` | Enable resources requests and limits for Pods. | `False` |
+| `pod.resources.nginx.requests` | Define resources requests and limits for Nginx Pods. | `{}` |
+| `pod.resources.magmalte.requests` | Define resources requests and limits for Magmalte Pods. | `{}` |
+| `proxy.nodeSelector` | Define which Nodes the Pods are scheduled on. | `{}` |
+| `proxy.tolerations` | If specified, the pod's tolerations. | `[]` |
+| `proxy.affinity` | Assign the orchestrator proxy to run on specific nodes. | `{}` |
+
+```
+- Install NMS chart:
+```bash
+$ cat vals.yaml
+magmalte:
+  env:
+    api_host: orc8r-proxy.magma.svc.cluster.local:9443
```

Comment:
Just to keep the vals.yaml file minimal, we wouldnt need to change api_host, host and port right? Shall we remove them from here?